qemu/qemu.go

```diff
@@ -2236,6 +2236,9 @@ type Config struct {
 	// fds is a list of open file descriptors to be passed to the spawned qemu process
 	fds []*os.File
 
+	// FwCfg is the -fw_cfg parameter
+	FwCfg string
```

Comment:
In order to support: `fw_cfg [name=]name,file=file` and `-fw_cfg [name=]name,string=str` we'll need a `struct`

```go
type FwCfg struct {
   Name string
   File string
   Str string
}
```

note that `File` and `Str` are mutually exclusive, so please add a comment

Reply:
Implemented as a struct